README.md

```diff
@@ -23,14 +23,23 @@ A minimalist Vim plugin manager.
 [startup-time]: http://junegunn.kr/images/vim-startup-time.png
 
 ### Installation
-
 [Download plug.vim](https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim)
+###### Unix
 and put it in ~/.vim/autoload
```

Comment:
Thanks, but I would rather not split the sentence since doing so will give the impression that manual downloading of the link is necessary. How about changing it to "Download plug.vim and put it in ~/.vim/autoload (or ~/vimfiles/autoload on Windows)?